slang.h

```diff
@@ -2988,6 +2988,22 @@ namespace slang
         virtual SLANG_NO_THROW void SLANG_MCALL getLanguagePrelude(
             SlangSourceLanguage sourceLanguage,
             ISlangBlob** outPrelude) = 0;
+
+        
+            /** Compile from (embedded source) the StdLib on the session.
+            Will return a failure if there is already a StdLib available
+            NOTE! API is experimental and not ready for production code 
+            */
+        virtual SLANG_NO_THROW SlangResult SLANG_MCALL compileStdLib() = 0;
```

Comment:
As already predicted, I'm not a huge fan of having these in the public API, and I'm even less of a fan of having them on a core interface like this (so that we are potentially stuck having them in the binary interface "forever").

Realistically, I don't expect these to create a binary-compatibility issue, and we can probably pull them in a later version/release without breaking anything.